nixos/modules/system/boot/stage-1.nix

```diff
@@ -226,16 +235,18 @@ let
         echo "testing patched programs..."
         $out/bin/ash -c 'echo hello world' | grep "hello world"
         ${
-          if zfsRequiresMountHelper then
+          if fsRequiresUtilLinuxMount then
             ''
               $out/bin/mount -V 1>&1 | grep -q "mount from util-linux"
-              $out/bin/mount.zfs -h 2>&1 | grep -q "Usage: mount.zfs"
             ''
           else
             ''
               $out/bin/mount --help 2>&1 | grep -q "BusyBox"
             ''
         }
+        ${optionalString zfsRequiresMountHelper ''
```

Comment:
doesn't need to be solved in this pull request. But I have the feeling that the zfs code should be moved into the zfs module.

Reply:
config.boot.initrd.extraUtilsCommandsTest

Reply:
we also do this for multipath...

Reply:
addressed by latest push 👍